### PR TITLE
Demo branch

### PR DESCRIPTION
**Pull request by jzoon**
A better version of our agent that we made for the demo. User story:
"As an developer
I want to give a demo 
so that it is clear to people
how our agent works at this point"

### CODE REVIEW

File: TU-Agent/dummy.pl [hunk 1]
```diff
@@ -4,14 +4,17 @@
...
+havebuilding :- buildings([X,Y,Z]).
 
 % demolish
 % demolish :- buildings(none).
 
 % initialize built believe
 built(none).
 
+% initialize destroy believe
+destroy(none).
```
Comment (daanwerf) [inline TU-Agent/dummy.pl:17]: Maybe you could replace "none" for a more descriptive text? Or else leave the brackets out completely.

**Comment by tvleest** [comment]
Is it necessary to merge the demo to the master?

**Comment by jzoon** [comment]
This isn't necessary, but this bot is slightly better then the bot in the master at this point. So we thought it would be a good idea to change the bot in the master to this bot.

**Comment by spiderbiggen** [comment]
Looks good. for newer versions please use a more up to date environment

**Comment by daanwerf** [comment]
I think it is fine to merge this branch into the master, as it is pretty much the same as the master but with a more functional bot. The fact that is was meant for the demo does not matter.

**Comment by jzoon** [comment]
Okay, I'll merge it

**Pull request merged**